hudi-common/src/main/java/org/apache/hudi/metadata/HoodieMetadataPayload.java

```diff
@@ -390,8 +390,12 @@ private HoodieMetadataColumnStats combineColumnStatsMetadata(HoodieMetadataPaylo
     return combineAndGetUpdateValue(oldRecord, schema, new Properties());
   }
 
+  public Option<IndexedRecord> getInsertValue() throws IOException {
+    return getInsertValue(null, null);
```

Comment:
whats the purpose of this? payload impls are not supposed to have any additional public methods. so trying to understand the use-case for this ?

Reply:
not really needed, i think we can pass `null` from the caller itself and mention that generating metadata record doesn't rally depend on schema